includes/Core/Authentication/Authentication.php

```diff
@@ -217,49 +245,6 @@ function( $routes ) {
 			}
 		);
 
-		add_filter(
-			'googlesitekit_inline_base_data',
-			function ( $data ) {
-				return $this->inline_js_base_data( $data );
-			}
-		);
-
-		add_filter(
-			'googlesitekit_admin_data',
-			function ( $data ) {
-				return $this->inline_js_admin_data( $data );
-			}
-		);
-
-		add_filter(
-			'googlesitekit_setup_data',
-			function ( $data ) {
-				return $this->inline_js_setup_data( $data );
-			}
-		);
-
-		add_filter(
-			'allowed_redirect_hosts',
-			function ( $hosts ) {
-				return $this->allowed_redirect_hosts( $hosts );
-			}
-		);
-
-		add_filter(
-			'googlesitekit_admin_notices',
-			function ( $notices ) {
-				return $this->authentication_admin_notices( $notices );
-			}
-		);
-
-		add_action(
-			'admin_action_' . Google_Proxy::ACTION_SETUP,
-			function () {
-				$this->verify_proxy_setup_nonce();
-			},
-			-1
-		);
-
 		add_action(
 			'admin_action_' . Google_Proxy::ACTION_SETUP,
```

Comment:
Just for code clarity, can we move this section to the other `'admin_action_' . Google_Proxy::ACTION_SETUP` hooks above (as the last one of them, since it runs last, priority 10)?

Reply:
Yep, moved.